src/odemis/gui/cont/acquisition.py

```diff
@@ -696,6 +694,30 @@ def _on_export_data_done(self, future):
         data = future.result()
         self._display_acquired_data(data)
 
+    def _add_feature_info_filename(self):
+        """
+        Add feature name, feature status and the counter at the end of the filename.
+        """
+        local_tab = self._tab_data.main.getTabByName("cryosecom-localization")
+        local_tab.tab_data_model.select_current_position_feature()
+        feature_pattern = (local_tab.tab_data_model.main.currentFeature.value.name.value +
+                 '-' + local_tab.tab_data_model.main.currentFeature.value.status.value)
```

Comment:
Small suggestion, do with it what you want:

This is not the nicest to read and it is hard to see what the pattern looks like, I would suggest something like:
```suggestion
        feature_name = local_tab.tab_data_model.main.currentFeature.value.name.value
        feature_status = local_tab.tab_data_model.main.currentFeature.status.name.value
        feature_pattern = f"{feature_name}-{feature_status}"
```